test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageControllerSpec.scala

```diff
@@ -3177,6 +3370,7 @@ class MultipleDisposalsTriageControllerSpec
     checkPageIsDisplayed(
       performAction(),
       messageFromMessageKey(expectedPageTitleMessageKey), { doc =>
+        println(doc.body().html())
```

Comment:
can you do a global search for `println` and remove them all?

Reply:
Done. There were 2. sorry for the mistake.